x-pack/plugins/lens/public/xy_visualization/xy_config_panel.tsx

```diff
@@ -529,22 +565,36 @@ export function DimensionEditor(props: VisualizationDimensionEditorProps<State>)
           idSelected={`${idPrefix}${axisMode}`}
           onChange={(id) => {
             const newMode = id.replace(idPrefix, '') as YAxisMode;
-            const newYAxisConfigs = [...(layer.yConfig || [])];
-            const existingIndex = newYAxisConfigs.findIndex(
-              (yAxisConfig) => yAxisConfig.forAccessor === accessor
-            );
-            if (existingIndex !== -1) {
-              newYAxisConfigs[existingIndex].axisMode = newMode;
-            } else {
-              newYAxisConfigs.push({
-                forAccessor: accessor,
-                axisMode: newMode,
-              });
-            }
+            const newYAxisConfigs = createNewYAxisConfigsWithValue('axisMode', newMode);
             setState(updateLayer(state, { ...layer, yConfig: newYAxisConfigs }, index));
           }}
         />
       </EuiFormRow>
+      {shouldShowValueLabels && (
+        <EuiFormRow
+          display="columnCompressedSwitch"
+          fullWidth
+          label={i18n.translate('xpack.lens.xyChart.showValueLabels.label', {
+            defaultMessage: 'Show Value Labels',
+          })}
+        >
```

Comment:
Similar to fitting functions, please add a tooltip with a sentence like "value labels can only be shown for bar charts with a single y-axis series"